src/kssh/flags.go

```diff
@@ -5,6 +5,9 @@ import "fmt"
 type CLIArgument struct {
 	Name        string // eg "--foo"
 	HasArgument bool   // true if an argument comes after it (eg "--foo bar") false if it is a boolean flag (eg "--help")
+	Preserve    bool   // true if you wish to preserve this argument into the list of remaining arguments even if found
+	// eg if your command takes in a `-v` flag and the subcommand also takes in a `-v` flag
+	// incompatible with HasArgument: true
```

Comment:
there's no specific reason this is true, right? just that's it's not been implemented? if so, suggest:

```suggestion
	// incompatible with `HasArgument: true` but only because it hasn't been built
```